.github/workflows/examples-components.yml

```diff
@@ -168,6 +168,11 @@ jobs:
         run: |
           npm install -g @bytecodealliance/jco
           npm install -g @bytecodealliance/componentize-js
+      # Validate example wadm.yaml if the example has one
+      - name: validate wadm.yaml
+        if: ${{ hashFiles(format('examples/{0}/{1}/components/wadm.yaml', matrix.project.lang, matrix.project.name)) != '' }}
```

Comment:
Blergh, I can't figure out what to change to make this work. We could opt to require wadm's for every component example. Thoughts?

Reply:
And also should it switch on local.wadm.yaml vs wadm.yaml?

Reply:
Here you can solve it by checking whether the file(s) are there or not in a meta step above! I'll commit something

Reply:
Let me know what you think about this: 

https://github.com/wasmCloud/wasmCloud/pull/2700/commits/8836dac192076f937f82063d755a647b9b4fc49c